### PR TITLE
Update en.js

### PR DESCRIPTION
**Pull request by paulhinz**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **kerruba**

If they are not already assigned, you can assign the PR to them by writing `/assign @kerruba` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/locales/en.js [hunk 1]
```diff
@@ -477,55 +477,55 @@ export default {
     'pageSettings.input.baseURL.static': 'Static',
     'pageSettings.input.appendBaseURL': 'Append context name on BaseURL',
     'pageSettings.input.jsession': 'Use JSESSIONID',
-    'pageSettings.input.languageBroswer': 'Home page gets its language from the browser',
-    'pageSettings.input.pageTreeStyle': 'Choose the style of the Page tree',
+    'pageSettings.input.languageBroswer': 'Home page localization is selection by the requesting browser',
```
Comment (gidesan) [inline src/locales/en.js:480]: Hey Paul, you meant "selected"?

**Review by gidesan — CHANGES_REQUESTED** [review]

File: src/locales/en.js (hunk 1 above)
Comment (paulhinz) [reply src/locales/en.js:480]: Agree   - "selected"

**Review by gidesan — APPROVED** [review]

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/785#pullrequestreview-455606075" title="Approved">gidesan</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)~~ [gidesan]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**